python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -76,17 +76,21 @@ def to_file(j, fobj=None, ignore_subs=''):
 # * AssertionError (corruption: multiple objects in <root>...</root>
 # * Exception (probably corrupted data problem)
 
-def _raw_from_file(f):
+def _raw_from_file(f, obj):
     # logger.debug('----------------------------')
     ###logger.debug('Parsing file: %s',f.name)
     xml_content = f.read()
-    obj, errors = Loader().parse(xml_content)
+    xml_lines = xml_content.splitlines()
+    for line in xml_lines:
+        line = line.lstrip()
+    xml_content = '\n'.join(xml_lines)
+    obj, errors = Loader(obj).parse(xml_content)
```

Comment:
I'm looking at the case of files with ~ 10,000 GangaObjects in them so I'm moving to avoid creating too many new objects as and when they're required in favor of doing things in place